src/module.c

```diff
@@ -2311,6 +2311,20 @@ RedisModuleString *RM_CreateStringFromLongLong(RedisModuleCtx *ctx, long long ll
     return RM_CreateString(ctx,buf,len);
 }
 
+/* Like RedisModule_CreatString(), but creates a string starting from a `unsigned long long`
```

Comment:
There's a typo where you copied this. It's the spelling of `RedisModule_CreateString`.

```suggestion
/* Like RedisModule_CreateString(), but creates a string starting from a `unsigned long long`
```

Reply:
FYI, all existing occurrences of this typo are fixed in this PR: #10890.

Reply:
Sorry, I didn't note this.